modelforge/utils/units.py

```diff
@@ -43,6 +43,29 @@
 )
 
 
+def _convert_str_to_unit_length(val: Union[unit.Quantity, str]) -> float:
+    """Convert a string representation of an OpenFF unit to a unit.Quantity
+    If the input is already a unit.Quantity, it is returned as is.
```

Comment:
Can you update the doc string to reflect this converts to nanometers and removes units?